dashboard/app/auth_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/google/syzkaller/dashboard/dashapi"
```

Comment:
Move this to the bottom and add a blank line. We separate non-standard packages.